source/supported-hardware/raptor-lake.rst

```diff
@@ -0,0 +1,187 @@
+  +-------------------------+---------------+----------------+---------------+---------------+
+  | |RPLS| DDR5 (SODIMM S17)|     0x0011    |      J1G2      |     R6J1      |     rpls      |
+  +-------------------------+---------------+----------------+---------------+---------------+
+
+Debug UART
+^^^^^^^^^^^
+
+For |RPL| platforms, serial port connector location can be found from the above table for each supported target board.
+
+.. note:: Configure host PuTTY or minicom to 115200bps, 8N1, no hardware flow control.
+
+Building
+^^^^^^^^^^
+
+To build |SPN| for any |RPL| platform::
+
+    python BuildLoader.py build <PLAT>
+
+    <PLAT> = rpls
+
+The output images are generated under ``Outputs`` directory.
+
+
+Stitching
+^^^^^^^^^^
+
+1. Gather |RPL| IFWI firmware image
+
+  Users can either download the full IFWI image if the IFWI image release is available or read the existing IFWI image on the board using SPI programmer.
+  This image contains additional firmware ingredients that are required boot on |RPL|.
+
+.. note::
+  ``StitchLoader.py`` currently does not support stitching with boot guard feature **enabled**.
+  To stitch with Boot Guard enabled, please use ``StitchIfwi.py``.
+
+
+2. Stitch |SPN| images into downloaded BIOS image::
+
+    python Platform/RaptorlakeBoardPkg/Script/StitchLoader.py -i <BIOS_IMAGE_NAME> -s Outputs/<plat>/SlimBootloader.bin -o <SBL_IFWI_IMAGE_NAME>
+
+  where -i = Input file, -o = Output file, plat = rpls
+
+For example, to stitch |SPN| IFWI image ``sbl_rpls_ifwi.bin`` from |RPLS| downloaded firmware images::
+
+    python Platform/RaptorlakeBoardPkg/Script/StitchLoader.py -i xxxx.bin -s Outputs/rpls/SlimBootloader.bin -o sbl_rpls_ifwi.bin
+
+For more details on stitch tool, see :ref:`stitch-tool` on how to stitch the IFWI image with |SPN|.
+
+
+Flashing
+^^^^^^^^^
+
+Flash the generated ``sbl_rpls_ifwi.bin`` to the target board using a DediProg SF100 or SF600 programmer.
+
+.. note:: Refer the table above to identify the connector on the target board for SPI flash programmer. When using such device, please ensure:
+
+
+    #. The alignment/polarity when connecting Dediprog to the board. 
+    #. The power to the board is turned **off** while the programmer is connected (even when not in use).
+    #. The programmer is set to update the flash from offset 0x0.
+
+
+Capsule image for |RPL|
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+|SPN| binary for capsule image could be extracted from |SPN| IFWI image generated in above steps.
+
+Extract ``bios.bin`` from |SPN| IFWI image::
+
+  python BootloaderCorePkg/Tools/IfwiUtility.py extract -i sbl_rpls_ifwi.bin -p IFWI/BIOS -o bios.bin
+
+Generate capsule update image ``FwuImage.bin``::
+
+  python BootloaderCorePkg/Tools/GenCapsuleFirmware.py -p BIOS bios.bin -k <Keys> -o FwuImage.bin
+
+.. note::
+BootGuard would not work if directly using Outputs/<plat>/SlimBootloader.bin for capsule image.
```

Comment:
Sphinx build error:
source\supported-hardware\raptor-lake.rst:94: ERROR: Content block expected for the "note" directive; none found.
source\supported-hardware\raptor-lake.rst:95: WARNING: Explicit markup ends without a blank line; unexpected unindent.
I think line 95 needs to be appended to the end of the "note::" on line 94.

Reply:
Thanks for the comments. it is updated.